src/cmd/prune.ts

```diff
@@ -0,0 +1,9 @@
+import prune from '../api/prune'
+import {PublicInstallationOptions} from '../api/install'
+
+export default (input: string[], opts: PublicInstallationOptions) => {
+  if (input && input.length) {
+    return prune(input, opts)
+  }
+  return prune(opts)
```

Comment:
i think that if you arent concerned about opts, it doesnt hurt to pass `{}`. it will be just more conscious usage for us or consumers.